tools/unicode_properties_parse/grapheme_break_property_data_gen.py

```diff
@@ -244,10 +244,10 @@ def generate_data_tables() -> str:
     GraphemeBreakProperty.txt and emoji-data.txt.
 
     GraphemeBreakProperty.txt can be found at
-    https://www.unicode.org/Public/14.0.0/ucd/auxiliary/GraphemeBreakProperty.txt
+    https://www.unicode.org/Public/UCD/latest/ucd/auxiliary/GraphemeBreakProperty.txt

```

Comment:
I'm slightly uncomfortable with non-permalinks, because it makes it harder to tell when the files were last regenerated. It also makes programmer-archaeology more difficult (what versions of Unicode did VS 2077 and VS 2097 use?).

I wonder, could we capture the first two comments from each file in the generated output? For these latest files, they are:

```
# GraphemeBreakProperty-15.0.0.txt
# Date: 2022-04-27, 17:07:38 GMT

# emoji-data.txt
# Date: 2022-08-02, 00:26:10 GMT

# GraphemeBreakTest-15.0.0.txt
# Date: 2022-02-26, 00:38:37 GMT
```

(14.0 said `emoji-data-14.0.0.txt` so it looks like 15.0 inadvertently omitted the version. At least they have the timestamp.)

If we captured and transformed these into `// comments` then that would resolve my concern.

Reply:
OK, now script captures first two comments

```text
# GraphemeBreakProperty-15.0.0.txt
# Date: 2022-04-27, 17:07:38 GMT
```

and appends them before enum and array definitions (https://github.com/microsoft/STL/pull/3556/commits/11214b049c01be113352c77c1d0d38beac5541b0).